### PR TITLE
chore: Move to full release

### PR DESCRIPTION
**Pull request by mowies**
### This PR
- removes release-please settings so that the next version will be 1.0.0
- removes the config file for standard-version which is not needed anymore

### CODE REVIEW

File: release-please-config.json [hunk 1]
```diff
@@ -6,8 +6,6 @@
         "changelog-path": "CHANGELOG.md",
         "release-type": "go",
         "prerelease": false,
```
Comment (philipp-hinteregger) [inline release-please-config.json:8]:
```suggestion
```

Comment (philipp-hinteregger) [reply release-please-config.json:8]: I think we don't need this anymore too

**Review by philipp-hinteregger — CHANGES_REQUESTED** [review]

**Review by philipp-hinteregger — APPROVED** [review]
LGTM

**Review by thisthat — CHANGES_REQUESTED** [review]
Looking forward to merge this PR :D 
I am asking for changes just to block it until we are fully ready to officialize the LTS release

**Review by thisthat — APPROVED** [review]
All pieces for Keptn v1.0.0 have been merged 🎉

**Pull request merged**